package.json

```diff
@@ -45,6 +48,8 @@
     "eslint": "^7.32.0",
     "eslint-import-resolver-typescript": "^2.4.0",
     "fetch-mock": "^9.11.0",
+    "global-jsdom": "^8.2.0",
```

Comment:
I'm more familiar with [`jsdom-global`](https://www.npmjs.com/package/jsdom-global) which is a bit more popular as well (based on weekly downloads), but it appears they do the same thing anyways 🤷

Reply:
I copied these docs:
https://testing-library.com/docs/react-testing-library/setup/#using-without-jest

Reply:
Ah, In that case, I see no reason to change 👍